Overview.src.html

```diff
@@ -4167,40 +4235,90 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 <ol>
  <li><p>Let <var title>p</var> be a new promise.
 
- <li><p>Let <var title>r</var> be the associated
+ <li><p>Let <var title>request</var> be the associated
  <span title=concept-request>request</span> of the result of invoking the initial value of
  <code title=dom-Request>Request</code> as constructor with <var title>input</var> and
  <var title>init</var> as arguments. If this throws an exception, reject
- <var title>p</var> with it.
+ <var title>p</var> with it and return <var title>p</var>.
 
- <li><p>Set <var title>r</var>'s <span title=concept-request-initiator>initiator</span> to
+ <li><p>Set <var title>request</var>'s <span title=concept-request-initiator>initiator</span> to
  "<code title>fetch</code>".
 
- <li><p>Set <var title>r</var>'s
+ <li><p>Set <var title>request</var>'s
  <span title=concept-request-destination>destination</span> to
  "<code title>subresource</code>".
 
+ <li><p>Let <var title>stream</var> be null.
+
  <li>
   <p>Run the following <span data-anolis-spec=html>in parallel</span>:
 
-  <p><span title=concept-fetch>Fetch</span> <var title>r</var>.
+  <p><span title=concept-fetch>Fetch</span> <var title>request</var>.
 
   <p>To <span>process response</span> for <var title>response</var>, run these substeps:
 
   <ol>
    <li><p>If <var title>response</var>'s <span title=concept-response-type>type</span> is
    "<code title>error</code>", reject <var title>p</var> with a
-   <code title>TypeError</code>.
+   <code title>TypeError</code> and abort these steps.
 
-   <li><p>Otherwise, resolve <var title>p</var> with a new <code>Response</code> object
-   associated with <var title>response</var> and a new <code>Headers</code> object whose
+   <li><p>Let <var title>res</var> be a new <code>Response</code> object associated with
+   <var title>response</var> and a new <code>Headers</code> object whose
    <span title=concept-Headers-guard>guard</span> is "<code title>immutable</code>".
+
+   <li><p>If <var title>response</var>'s status is a <span>null body status</span>, resolve
+   <var title>p</var> with <var title>res</var> and abort these steps.
```

Comment:
This is related to the below comment. If &lt;concept-response-body&gt; is a ReadableStream, we don't need this item. Otherwise, we need this early-exit to make `res` have null body.